definitions/control-v1.yaml

```diff
@@ -1121,51 +1123,52 @@ paths:
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '404':
           description: App not found
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '500':
           description: Internal server error
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
+                '$ref': '#/components/schemas/error'
         '504':
           description: Gateway timeout
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/error"
-  "/me":
+                '$ref': '#/components/schemas/error'
+  '/me':
     get:
       summary: Get token details
+      description: Retrieve information about the <a href="https://ably.com/documentation/core-features/authentication#token-authentication">token</a> the current user authenticates with.
```

Comment:
Perhaps it would be better to link to here: https://ably.com/documentation/control-api#authentication 

The reason is we're talking about Control API tokens specifically here.

Reply:
Good point, changed.